service/devdoc/service_amqp_requirements.md

```diff
@@ -88,6 +88,27 @@ The `send` method sends an event to the device passed as argument, using the IoT
 
 **SRS_NODE_IOTHUB_SERVICE_AMQP_16_030: [** The `send` method shall call the `send` method of the C2D link and pass it the Amqp request that it created. **]**
 
+### sendToModule(deviceId: string, moduleId: string, inputName: string, message: Message, done?: Callback<results.MessageEnqueued>): void;
+
+**SRS_NODE_IOTHUB_SERVICE_AMQP_18_001: [** The `sendToModule` method shall construct an AMQP request using the message passed in argument as the body of the message. **]**
+
+**SRS_NODE_IOTHUB_SERVICE_AMQP_18_002: [** The message generated by the `sendToModule` method should have its “to” field set to the `deviceId` & `moduleId` passed as arguments. **]**
```

Comment:
>set to the `deviceId` & `moduleId` passed as arguments. [](start = 121, length = 55)

add string formatting information.

Reply:
thanks!!

---
In reply to: [180599988](https://github.com/Azure/azure-iot-sdk-node/pull/271#discussion_r180599988) [](ancestors = 180599988)